### ISSUE TITLE
Is there a way to set render_warning = false for the plugin via perspective-workspace layout configuration ?

### ISSUE THREAD

**Issue by sv2** [finos/perspective#1963]
## How to set render_warning: false via perspective-workspace layout configuration ?

I'd like to configure perspective-workspace layout so that "Render all point" warning is not shown on the plugin with large dataset. 
Tried to pass render_warning: false via plugin_config, but unfortunately this has no effect:

```
  viewers: {
    METRICS: {
      plugin: 'Y Line',
      plugin_config: { render_warning: false, splitMainValues: ['rate'] },
      settings: false,
      group_by: ['ts'],
      split_by: [],
      columns: ['value', 'rate'],
      filter: [],
      sort: [],
      expressions: [],
      aggregates: {},
      name: 'metrics',
      table: 'datatable',
    },
  }
```
Using the latest version of perspective 

Is there any other way to specify that in the layout configuration ? 

Thanks in advance !

**Comment by texodus** [comment]
No, but you can change the limits globally, or mark `render_warning` as confirmed, [on the global instance of the plugin as returned by `customElements.get()`](https://github.com/finos/perspective/blob/master/examples/blocks/src/fractal/index.js#L146).
